phoenix-core/src/main/java/org/apache/phoenix/coprocessor/MetaDataEndpointImpl.java

```diff
@@ -2324,96 +2488,97 @@ else if (pkCount <= COLUMN_NAME_INDEX
-                            byte[] ptr = new byte[PInteger.INSTANCE.getByteSize()];
-                            PInteger.INSTANCE.getCodec().encodeInt(newPosition, ptr, 0);
-                            viewColumnPut.add(PhoenixDatabaseMetaData.TABLE_FAMILY_BYTES,
-                                    PhoenixDatabaseMetaData.ORDINAL_POSITION_BYTES, clientTimeStamp, ptr);
-                            mutationsForAddingColumnsToViews.add(viewColumnPut);
-                        } else {
-                            continue; // move on to the next column
-                        }
-                    } else {
-                        int newOrdinalPosition = p.ordinalPosition;
-                        /*
-                         * For a non-diverged view, we need to make sure that the base table column
-                         * is added at the right position.
-                         */
-                        if (ordinalPositionList.size() == 0) {
-                            ordinalPositionList.setOffset(newOrdinalPosition);
-                            ordinalPositionList.addColumn(columnKey, newOrdinalPosition);
-                            for (PColumn col : view.getColumns()) {
-                                int ordinalPos = getOrdinalPosition(view, col);
-                                if (ordinalPos >= newOrdinalPosition) {
-                                    // increment ordinal position of columns by 1
-                                    int updatedPos = ordinalPos + 1;
-                                    ordinalPositionList.addColumn(getColumnKey(viewKey, col), updatedPos);
-                                } 
-                            }
-                        } else {
-                            ordinalPositionList.addColumn(columnKey, newOrdinalPosition);
-                        }
-                        mutationsForAddingColumnsToViews.add(viewColumnPut);
-                    }
-                    if (isPkCol) {
-                        deltaNumPkColsSoFar++;
-                        // Set the key sequence for the pk column to be added
-                        short currentKeySeq = SchemaUtil.getMaxKeySeq(view);
-                        short newKeySeq = (short)(currentKeySeq + deltaNumPkColsSoFar);
-                        byte[] keySeqBytes = new byte[PSmallint.INSTANCE.getByteSize()];
-                        PSmallint.INSTANCE.getCodec().encodeShort(newKeySeq, keySeqBytes, 0);
-                        viewColumnPut.add(PhoenixDatabaseMetaData.TABLE_FAMILY_BYTES,
-                                PhoenixDatabaseMetaData.KEY_SEQ_BYTES, keySeqBytes);
-                        addMutationsForAddingPkColsToViewIndexes(mutationsForAddingColumnsToViews, clientTimeStamp, view,
-                                deltaNumPkColsSoFar, columnName, viewColumnPut);
-                    }
-                    columnsAddedToView++;
-                    columnsAddedToBaseTable++;
                 }
+//                else {
```

Comment:
You'll likely get to this later, but let's remove the commented out code when you're ready.